pkg/messagebroker/kafka_test.go

```diff
@@ -43,11 +43,30 @@ func Test_Pause(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func Test_DescribeTopicConfigs(t *testing.T) {
+	a := getKafkaAdmin()
+	resp, err := a.DescribeTopicConfigs(context.Background(), []string{})
+	assert.NotNil(t, err)
+	assert.Nil(t, resp)
+}
+
+func Test_AlterTopicConfigs(t *testing.T) {
+	a := getKafkaAdmin()
+	resp, err := a.AlterTopicConfigs(context.Background(), ModifyTopicConfigRequest{TopicConfigs: nil})
+	assert.NotNil(t, err)
```

Comment:
@sakshi-goyal-razorpay we are asserting not nil for err, should we also add a nil test case as well?

Reply:
@akashgoyal96 Its a unit test. It won't actually be able to get to kafka broker to alter or describe configs. Those cases are added as part of integration test